connection.js

```diff
@@ -643,44 +781,66 @@ function customBoardsizeWidthsHeightsReject(argNameString, notificationWidth, no
-}  /* }}} */
+function familyArrayFromFamilyNameString(familyNameString) { /* {{{ */
+    return ["", "ranked", "unranked"].map(e => familyNameString + e);
+} /* {{{ */
 
-// minmax families reject functions :
-function minmaxPeriodsActualReject(familyNameString, familyNotification, notificationTSpeed, notificationRanked) { /* }}} */
-    /* "fischer", "simple", "absolute", "none", don't have a periods number,
-    /  so this function only applies to "byoyomi" and "canadian"*/
-    for (let blitzLiveCorr of ["blitz", "live", "corr"]) {
-        if (notificationTSpeed === convertBlitzLiveCorr(blitzLiveCorr)) {
-            const minFamilyObject = familyObjectMIBLIsminmax("min" + familyNameString + blitzLiveCorr);
-            const maxFamilyObject = familyObjectMIBLIsminmax("max" + familyNameString + blitzLiveCorr);
-            /* example : {argNameStrings {all: "minperiodsblitz", ranked: "minperiodsblitzranked", unranked: "minperiodsblitzunranked"},
-                                    MIBL {minMax: mm, incDec: ir, belAbo: ba, lowHig: lh},
-                                    isMM {isMin: true, isMax: false}};*/
-            let argNameString = "";
-            for (let familyObject of [minFamilyObject, maxFamilyObject]) {
-                argNameString = checkObjectArgsToArgNameString(familyObject.argNameStrings, notificationRanked);
-                if (minmaxCondition(config[argNameString], familyNotification, familyObject.isMM.isMin)) { // if we dont reject, we early exit all the remaining reject
-                    const rankedUnranked = beforeRankedUnrankedGamesSpecial("for ", notificationTSpeed + " ", argNameString, "");
-                    conn_log(`${familyNotification} is ${familyObject.MIBL.belAbo} ${familyObject.MIBL.minMax} ${familyNameString} ${rankedUnranked} ${config[argNameString]}`);
-                    return { reject: true, msg: `${familyObject.MIBL.minMax} ${familyNameString} ${rankedUnranked} ${config[argNameString]}, please ${familyObject.MIBL.incDec} the number of ${familyNameString}.` };
-                }
-            }
-        }
+function familyObjectMIBL(familyNameString) { /* {{{ */
+    let mm = "";
+    let ir = "";
+    let ba = "";
+    let lh = "";
+    const isMin = familyNameString.includes("min");
+    const isMax = familyNameString.includes("max");
+    if (isMin) {
+        mm = "Minimum";
+        ir = "increase";
+        ba = "below";
+        lh = "low";
+    } else {
+        mm = "Maximum";
+        ir = "reduce";
+        ba = "above";
+        lh = "high";
     }
-}
+    const familyArray = familyArrayFromFamilyNameString(familyNameString);
+    return {argNameStrings: {all: familyArray[0], ranked: familyArray[1], unranked: familyArray[2]},
+                      MIBL: {minMax: mm, incDec: ir, belAbo: ba, lowHig: lh},
```

Comment:
If you rename `mm` to `minMax`, `ir` to `incDev` etc. you can use ` MIBL: {minMax, incDec, belAbo, lowHig},`.

Reply:
yes :100: